src/todt.c

```diff
@@ -62,15 +60,15 @@ Symbol *toSymbolCppTypeInfo(ClassDeclaration *cd);
 
 /* ================================================================ */
 
-dt_t **Initializer_toDt(Initializer *init, dt_t **pdt)
+void Initializer_toDt(Initializer *init, DtBuilder& dtb)
 {
     class InitToDt : public Visitor
     {
     public:
-        dt_t **pdt;
+        DtBuilder dtb;
 
-        InitToDt(dt_t **pdt)
-            : pdt(pdt)
+        InitToDt(DtBuilder& dtb)
+            : dtb(dtb)
```

Comment:
I know. Make it a DtBuilder* and it will work fine. (But on the C++ side, I'd have to rewrite every use of `dtb`.)